Makefile

```diff
@@ -0,0 +1,17 @@
+RM=rm -rf
+
+.PHONY: all clean
+
+all:
+	yarn install
+	yarn workspace button build
+
+watch:
+	yarn workspace $(P) watch
+
+build:
+	yarn workspace $(P) build
+
+clean:
+	$(RM) yarn.lock node_modules
```

Comment:
We probably do not want the lockfile and node_modules folders in a typical clean command that gets run. This might be better to call this something like a `force-clean` or something like that. This way we know it is a higher level command than a typical clean.

We will definitely want a clean command that does things like cleanup on runtimes from packages as well as build folders, which should not touch lockfile or modules folders.

Reply:
@lordpixel nevermind. I am resolving these issues. We can clean those things up separately now that this file is here.